test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -8,6 +8,8 @@
 using System.Threading.Tasks;
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.TestHost;
+using Microsoft.Framework.DependencyInjection;
```

Comment:
the test exists just to make sure we don't totally break the sample.  the sample exists just to show off a few tag helpers.
note my issue was not at all the use of the new tag helper in the sample.  I'm more worried about the inability to do what you're doing here with a site running out of process.  but if this is just  temporary pain to fake out the samples `Startup` class, might be fine.